can/interfaces/vector/canlib.py

```diff
@@ -217,20 +224,36 @@ def __init__(
         if permission_mask.value == self.mask:
             if fd:
                 self.canFdConf = xlclass.XLcanFdConf()
-                if bitrate:
-                    self.canFdConf.arbitrationBitRate = ctypes.c_uint(bitrate)
+                if timing:
```

Comment:
Can we refactor this out of the `init` and get back the `canFdConf` object? That should make the code more readible and make testing the corner cases a bit easier.

Reply:
@hardbyte can you take another look? I implemented the changes you requested.